### PR TITLE
PEP8 and style improvements

### PR DESCRIPTION
**Pull request by pietrodn**
Ran `flake8` on the code base and fixed most of the issues, with this patch

```
flake8 --ignore=E501,F811,E722,E262,E221 --exclude=dronekit/lib.py dronekit
```

succeeds without errors.
I also implemented some of the suggestions proposed by PyCharm's code inspection tools (chained comparisons and a documentation inconsistency).

### CODE REVIEW

File: dronekit/__init__.py [hunk 1]
```diff
@@ -1022,7 +1013,7 @@ def __init__(self, handler):
 
         # Cache all updated attributes for wait_ready.
         # By default, we presume all "commands" are loaded.
-        self._ready_attrs = set(['commands'])
+        self._ready_attrs = {'commands'}
```
Comment (peterbarker) [inline dronekit/__init__.py]: This I don't understand.  Moving from a set to a dictionary is strange.

File: dronekit/__init__.py [hunk 2]
```diff
@@ -1345,8 +1335,7 @@ def listener(_):
 
             # Timeouts.
             if self._heartbeat_started:
-                if self._heartbeat_error and self._heartbeat_error > 0 and monotonic.monotonic(
-                ) - self._heartbeat_lastreceived > self._heartbeat_error:
+                if self._heartbeat_error and monotonic.monotonic() - self._heartbeat_lastreceived > self._heartbeat_error > 0:
```
Comment (peterbarker) [inline dronekit/__init__.py]: break this to avoid too-long-a-line?

File: dronekit/__init__.py [hunk 3]
```diff
@@ -2084,12 +2071,12 @@ def get_alt():
 
         def check_alt():
             cur = get_alt()
-            delta = abs(alt-cur)
+            delta = abs(alt - cur)
 
             return (
                 (delta < epsilon) or
-                (start < alt and cur > alt) or
```
Comment (peterbarker) [inline dronekit/__init__.py]: This obfuscates what's going on, I think.

File: dronekit/__init__.py [hunk 4]
```diff
@@ -3019,6 +3004,8 @@ def connect(ip,
     :returns: A connected vehicle of the type defined in ``vehicle_class`` (a superclass of :py:class:`Vehicle`).
     """
 
+    from dronekit.mavlink import MAVConnection
```
Comment (peterbarker) [inline dronekit/__init__.py]: If we're changing this - why are we still polluting the namespace?

File: dronekit/mavlink.py [hunk 5]
```diff
@@ -58,23 +55,23 @@ def __init__(self, device, baud=None, input=True, broadcast=False, source_system
...
                 data, new_addr = self.port.recvfrom(65535)
             except socket.error as e:
-                if e.errno in [ errno.EAGAIN, errno.EWOULDBLOCK, errno.ECONNREFUSED ]:
+                if e.errno in [errno.EAGAIN, errno.EWOULDBLOCK, errno.ECONNREFUSED]:
                     return ""
             if self.udp_server:
                 self.addresses.add(new_addr)
             elif self.broadcast:
-                self.addresses = set([new_addr])
+                self.addresses = {new_addr}
```
Comment (peterbarker) [inline dronekit/mavlink.py]: Again - moving to dict from set?

File: dronekit/__init__.py (hunk 1 above)
Comment (pietrodn) [reply dronekit/__init__.py]: It’s not a dict. It’s a set literal. https://stackoverflow.com/questions/36674083/why-is-it-possible-to-replace-sometimes-set-with

**Comment by peterbarker** [comment]
Merged ,thanks!

**Pull request merged**